include/ncasset/AssetTypes.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include "ncmath/Geometry.h"
+
+#include <array>
+#include <vector>
+
+namespace nc::asset
+{
+enum class AssetType
+{
+    AudioClip,
+    Cubemap,
```

Comment:
Will you change to CubeMap? We've settled on treating it as two words - the jury seems to be split on the net.

Reply:
Fixed